File: examples/user/sendPasswordResetEmail/go/sendPasswordResetEmail.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/user"
)

// Send an email with a verification code to reset password.
// Call "ResetPassword" endpoint once user provides the code.
func SendPasswordResetEmail() {
	userService := user.NewUserService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := userService.SendPasswordResetEmail(&user.SendPasswordResetEmailRequest{
		Email:    "joe@example.com",
		FromName: "Awesome Dot Com",
		Subject:  "Password reset",
		TextContent: `Hi there,
 click here to reset your password: myapp.com/reset/code?=$code`,
	})
	fmt.Println(rsp, err)
}

File: examples/user/sendPasswordResetEmail/node/sendPasswordResetEmail.js
const { UserService } = require("micro-js-client/user");

// Send an email with a verification code to reset password.
// Call "ResetPassword" endpoint once user provides the code.
async function sendPasswordResetEmail() {
  let userService = new UserService(process.env.MICRO_API_TOKEN);
  let rsp = await userService.sendPasswordResetEmail({
    email: "joe@example.com",
    fromName: "Awesome Dot Com",
    subject: "Password reset",
    textContent:
      "Hi there,\n click here to reset your password: myapp.com/reset/code?=$code",
  });
  console.log(rsp);
}

sendPasswordResetEmail();
